src/main/scala/org/broadinstitute/dsde/firecloud/webservice/CookieAuthedApiService.scala

```diff
@@ -29,23 +29,25 @@ trait CookieAuthedApiService extends HttpService with PerRequestCreator with Fir
     // Note that this endpoint works in the same way as ExportEntitiesApiService tsv download.
     path( "cookie-authed" / "workspaces" / Segment / Segment/ "entities" / Segment / "tsv" ) {
       (workspaceNamespace, workspaceName, entityType) =>
-        formFields('FCtoken, 'attributeNames.?) { (tokenValue, attributeNamesString) =>
-          post { requestContext =>
-            val attributeNames = attributeNamesString.map(_.split(",").toIndexedSeq)
-            val userInfo = dummyUserInfo(tokenValue)
-            val exportArgs = ExportEntitiesByTypeArguments(requestContext, userInfo, workspaceNamespace, workspaceName, entityType, attributeNames)
-            val exportProps: Props = ExportEntitiesByTypeActor.props(exportEntitiesByTypeConstructor, exportArgs)
-            actorRefFactory.actorOf(exportProps) ! ExportEntitiesByTypeActor.ExportEntities
-          }
+      post {
```

Comment:
Now correctly rejects GET etc with 405 Method Not Allowed in all cases.

Previously allowed GET with the correct form structure until a later rejection gave it 400 Bad Request.